source/blog/2018-10-24-release-notes.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title: October 2018 Release Notes — Version 18.10.0
+published: true
+post_author:
+  display_name: Neuman Vong
+  twitter: neuman
+tags: Releases, Finagle, Finatra, Util, Scrooge, TwitterServer
+---
+
+Here we are in the month of October, just a few weeks until creatures of
+fantasy roam the streets together with people dressed up as creatures of
+fantasy 👹.
+
+Speaking of the fantastic, we have new releases!
+
+[Finagle 18.10.0](https://github.com/twitter/finagle/releases/tag/finagle-18.10.0),
+[Finatra 18.10.0](https://github.com/twitter/finatra/releases/tag/finatra-18.10.0),
+[Scrooge 18.10.0](https://github.com/twitter/scrooge/releases/tag/scrooge-18.10.0),
+[TwitterServer 18.10.0](https://github.com/twitter/twitter-server/releases/tag/twitter-server-18.10.0),
+and [Util 18.10.0](https://github.com/twitter/util/releases/tag/util-18.10.0).
+
+### [Finagle](https://github.com/twitter/finagle/) ###
+
+#### Deprecations
+
+* finagle-core: Deprecation warnings have been removed from the 'status', 'onClose',
+  and 'close' methods on `c.t.f.t.Transport`, and added to the corresponding methods
+  on `c.t.f.t.TransportContext`. [9ae6d978](https://github.com/twitter/finagle/commit/9ae6d978c9b0c77309e7d199f6f71c6cc5d58586)
+
+#### Runtime Behavior Changes
+
+* finagle-netty3: Implementations for 'status', 'onClose', and 'close' methods have
+  been moved from `c.t.f.n.t.ChannelTransportContext` to `c.t.f.n.t.ChannelTransport`.
+  [9ae6d978](https://github.com/twitter/finagle/commit/9ae6d978c9b0c77309e7d199f6f71c6cc5d58586)
+
+### [Finatra](https://github.com/twitter/finatra/) ###
+
+#### Fixed
+
+* finatra-thrift: Set the bound `StatsReceiver` in the underlying Finagle `ThriftMux` server
+  in the `c.t.finatra.thrift.ThriftServer`. This prevented testing of underlying Finagle server
+  stats as the `InMemoryStatsReceiver` used by the `EmbeddedThriftServer` was not properly passed
+  all the way through the stack. [33d0524b](https://github.com/twitter/finatra/commit/33d0524b76f943c5661c0b21ff0d38c1d6563202)
+
+#### Changed
+
+* finatra-http, finatra-thrift: Make HTTP and Thrift StatsFitlers "Response Classification"
+  aware. [4085d40c](https://github.com/twitter/finatra/commit/4085d40cb7cb607101564bbc488f9a1daea56dff)
+
+* finatra-http, finatra-thrift: (BREAKING API CHANGE) Update the `DarkTrafficFilterModule` in
+  both HTTP and Thrift to allow for specifying further configuration of the underlying Finagle client.
+  This allows users the ability to set Finagle client concerns like ResponseClassification or other
+  configuration not expressed by the DarkTrafficFilterModule's API.
+
+  - Additionally, the Thrift `DarkTrafficFilterModule` has been updated to be ThriftMux only. For more information on mux see: [What is ThriftMux](https://twitter.github.io/finagle/guide/FAQ.html?highlight=thriftmux#what-is-thriftmux>).
```

Comment:
I think the `>` should be dropped from the section name. As is, it doesn't link to the "What is ThriftMux?" section.

Reply:
```suggestion
  - Additionally, the Thrift `DarkTrafficFilterModule` has been updated to be ThriftMux only. For more information on mux see: [What is ThriftMux](https://twitter.github.io/finagle/guide/FAQ.html?highlight=thriftmux#what-is-thriftmux).
```